cfme/tests/middleware/test_provider_crud.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from cfme import login
+from utils import providers
+
+login.login_admin()
+
+
+@pytest.fixture(scope='function')
+def hawkular_provider():
+    return providers.get_crud('hawkular')
```

Comment:
@psav I have made a small PR on @vnugent 's branch that should bring some of the expected utils.provider changes.